### PR TITLE
Added code for checkstyle

### PR DESCRIPTION
**Pull request by oblodgett**

### CODE REVIEW

File: agr_indexer/src/main/java/org/alliancegenome/indexer/indexers/curation/service/BaseDiseaseAnnotationService.java [hunk 1]
```diff
@@ -15,6 +15,19 @@
 import java.util.Set;
 import java.util.concurrent.atomic.AtomicBoolean;
 
+import org.alliancegenome.curation_api.model.entities.AGMDiseaseAnnotation;
+import org.alliancegenome.curation_api.model.entities.Allele;
+import org.alliancegenome.curation_api.model.entities.AlleleDiseaseAnnotation;
+import org.alliancegenome.curation_api.model.entities.DiseaseAnnotation;
+import org.alliancegenome.curation_api.model.entities.Gene;
+import org.alliancegenome.curation_api.model.entities.GenomicEntity;
+import org.alliancegenome.neo4j.repository.AlleleRepository;
```
Comment (devopsagr) [inline agr_indexer/src/main/java/org/alliancegenome/indexer/indexers/curation/service/BaseDiseaseAnnotationService.java:18]: 🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.RedundantImportCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Duplicate import to line 7 - org.alliancegenome.neo4j.repository.AlleleRepository.

File: agr_indexer/src/main/java/org/alliancegenome/indexer/indexers/curation/service/BaseDiseaseAnnotationService.java [hunk 2]
```diff
@@ -15,6 +15,19 @@
...
+import org.alliancegenome.neo4j.repository.GeneRepository;
```
Comment (devopsagr) [inline agr_indexer/src/main/java/org/alliancegenome/indexer/indexers/curation/service/BaseDiseaseAnnotationService.java:19]: 🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.RedundantImportCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Duplicate import to line 8 - org.alliancegenome.neo4j.repository.GeneRepository.

File: agr_indexer/src/main/java/org/alliancegenome/indexer/indexers/curation/service/BaseDiseaseAnnotationService.java [hunk 3]
```diff
@@ -15,6 +15,19 @@
...
+import org.apache.commons.collections4.CollectionUtils;
```
Comment (devopsagr) [inline agr_indexer/src/main/java/org/alliancegenome/indexer/indexers/curation/service/BaseDiseaseAnnotationService.java:20]: 🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.RedundantImportCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Duplicate import to line 9 - org.apache.commons.collections4.CollectionUtils.

File: agr_indexer/src/main/java/org/alliancegenome/indexer/indexers/curation/service/BaseDiseaseAnnotationService.java [hunk 4]
```diff
@@ -15,6 +15,19 @@
...
+
+import lombok.extern.log4j.Log4j2;
```
Comment (devopsagr) [inline agr_indexer/src/main/java/org/alliancegenome/indexer/indexers/curation/service/BaseDiseaseAnnotationService.java:22]: 🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.RedundantImportCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Duplicate import to line 3 - lombok.extern.log4j.Log4j2.

File: agr_indexer/src/main/java/org/alliancegenome/indexer/indexers/curation/service/BaseDiseaseAnnotationService.java [hunk 5]
```diff
@@ -15,6 +15,19 @@
...
+import net.nilosplace.process_display.util.ObjectFileStorage;
```
Comment (devopsagr) [inline agr_indexer/src/main/java/org/alliancegenome/indexer/indexers/curation/service/BaseDiseaseAnnotationService.java:23]: 🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.RedundantImportCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Duplicate import to line 4 - net.nilosplace.process_display.util.ObjectFileStorage.

File: agr_api/src/main/java/org/alliancegenome/api/controller/DiseaseController.java [hunk 6]
```diff
@@ -1,12 +1,17 @@
...
-import com.fasterxml.jackson.core.JsonProcessingException;
-import com.fasterxml.jackson.databind.ObjectMapper;
-import io.quarkus.logging.Log;
-import jakarta.enterprise.context.RequestScoped;
-import jakarta.inject.Inject;
-import jakarta.ws.rs.core.MediaType;
-import jakarta.ws.rs.core.Response;
+import static org.alliancegenome.api.service.EntityType.DISEASE;
+import static org.alliancegenome.api.service.EntityType.GENE;
+import static org.alliancegenome.neo4j.entity.SpeciesType.YEAST;
```
Comment (devopsagr) [inline agr_api/src/main/java/org/alliancegenome/api/controller/DiseaseController.java]: 🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.UnusedImportsCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unused import - org.alliancegenome.neo4j.entity.SpeciesType.YEAST.

**Review by adamgibs — APPROVED** [review]

**Pull request merged**